Core/SubscriptionManager/src/main/resources/application.yml

```diff
@@ -4,6 +4,10 @@ server:
 
 atcontext:
   url: http://localhost:9090/ngsi-ld/contextes/
+query:
```

Comment:
We have already added in yml file when we had start to worked on mqtt support code in subscription manager.

Reply:
Oh ok didn't see that one. Then i'll close this and we can solve it with the mqtt pull request.